packages/vuetify/src/directives/resize/index.ts

```diff
@@ -1,11 +1,11 @@
-import { VNodeDirective } from 'vue/types/vnode'
+import { DirectiveBinding, ObjectDirective } from 'vue'
 
-interface ResizeVNodeDirective extends VNodeDirective {
-  value?: () => void
+interface ResizeDirectiveBinding extends DirectiveBinding {
+  value: () => void | undefined
```

Comment:
why is `undefined` necessary?

Reply:
value can't have `?` in Vue 3.

Reply:
Ah, I thought it's fn returning void or undefined, maybe a good idea to add parens here?

Reply:
will check using ts playground. I'm unsure about the order here.

Reply:
Even if it works parens could be added to avoid confusion

Reply:
Thanks! Parens here are must-have things!

![image](https://user-images.githubusercontent.com/19504461/73647865-71b14f00-468d-11ea-9988-48e2f4228a3e.png)

```suggestion
  value: (() => void) | undefined
```